examples/with_winit/src/multi_touch.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2023 the Vello Authors
+// Copyright 2021 the egui Authors and Vello Authors
```

Comment:
The name used in egui's (MIT) [license](https://github.com/emilk/egui/blob/5cf99c6308e36b5f36861accea6835f190924cce/LICENSE-MIT) is:

> Copyright (c) 2018-2021 Emil Ernerfeldt <emil.ernerfeldt@gmail.com>

Maybe we should ask the author for their preference?

Reply:
The file in question was mostly written by [Ivo Vollrath](https://github.com/quadruple-output) as seen in [egui#306](https://github.com/emilk/egui/pull/306). Beyond that, there have been other contributors.

Given that there are multiple contributors and that the original file doesn't have a header at all, I think it's fine to reference them as *egui Authors*.